sdk/test-utils/recorder/CHANGELOG.md

```diff
@@ -1,33 +1,24 @@
 # Release History
```

Comment:
Please do "Hide whitespace" to view better.
![image](https://user-images.githubusercontent.com/10452642/162849351-a35da863-08df-45ad-9f22-14a9e2dbd55d.png)